deployment-k8s/aws-s3-k8s/README.md

```diff
@@ -0,0 +1,100 @@
+<!--
+ Copyright 2018-2021 Crown Copyright
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# AWS Kubernetes Examples
+
+This example demonstrates different users querying an avro file over a REST api running within an Amazon Web Services (AWS) environment in Docker/Kubernetes containers.
```

Comment:
Querying a file does not sound right.  How about querying an avro resource over a REST api or something along those lines.